pr_1/javascript/random_samples/leftover_debugging.js

```
@@ -0,0 +1,24 @@
+
+// {fact rule=insecure-connection@v1.0 defects=1}
+// ruleid: javascript-confirm
+if ( confirm("pushem!") == true) {
+    r = "x";
+} else {
+    // {/fact}
+
+    // {fact rule=insecure-connection@v1.0 defects=1}
+    r = "Y";
+    // ruleid: javascript-debugger
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)